pkg/agent/baker.go

```diff
@@ -928,7 +929,9 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			return config.SSHStatus == datamodel.SSHOff
 		},
 		"GetSysctlContent": func() (string, error) {
-			sysctlTemplate, err := template.New("sysctl").Parse(sysctlTemplateString)
+			templateFuncMap := make(template.FuncMap)
+			templateFuncMap["getPortRangeEndValue"] = getPortRangeEndValue
```

Comment:
why define this as a template func and not call the go func directly? we do this in many places for e.g. gpu driver version, you don't need extra indirection -- that's what the map of template funcs already provides.

Reply:
oh jk, it's not the same template